init-data.h

```diff
@@ -2,9 +2,10 @@
 #define __INIT_DATA__
 
 
-#define SIZE 9u
+#define DATA_SIZE 9u
+#define DATA_SIZE_H DATA_SIZE/2
```

Comment:
What does _H suffix mean?
Maybe more obvious name - DATA_SIZE_MID or something like that?

Reply:
H means half :)
Ok I'll think about better name

Reply:
can you do this yourself after you merge this branch to yours, pls